Pipfile.lock

```diff
@@ -0,0 +1,196 @@
+{
```

Comment:
In a pretty active development environment this file may be volatile, so I'll admit I'm not entirely sure of the consequences of including/excuding from VCS. 

That said, it is suggested by the Pipenv team to include so that's what I've done here:

https://github.com/pypa/pipenv/issues/598